Capstone/Friends/FriendDetailsViewController.m

```diff
@@ -0,0 +1,128 @@
+//
+//  FriendDetailsViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/1/22.
+//
+#import <Parse/Parse.h>
+#import "ParseCollege.h"
+#import "Parse/PFImageView.h"
+#import "FriendDetailsViewController.h"
+#import "FriendMatchesCell.h"
+#import "DetailsViewController.h"
+ 
+@interface FriendDetailsViewController ()<UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UILabel *username;
+@property (weak, nonatomic) IBOutlet UIButton *sendFriendRequest;
+@property (weak, nonatomic) IBOutlet UIButton *cancelFriendRequest;
+@property (weak, nonatomic) IBOutlet UIImageView *userImage;
+@property (weak, nonatomic) IBOutlet UIButton *friends;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
```

Comment:
To decrease confusion between UI elements vs. data structures, we want to name IBOutlets with {name}{outlet name}. For example, `usernameLabel`, `sendFriendRequestButton`, etc.

Also, looks like we have a UIButton named `friends` and an NSMutableArray also named `friends` 😱

Reply:
resolved :)